cider/tests/utils/QueryDataGenerator.h

```diff
@@ -129,12 +196,11 @@ class QueryDataGenerator {
         case ::substrait::Type::KindCase::kFixedChar:
           GENERATE_AND_ADD_VARCHAR_COLUMN(CiderByteArray)
         case ::substrait::Type::KindCase::kDate:
-          GENERATE_AND_ADD_TIMING_COLUMN(CiderDateType)
-        // FIXME(jikunshang): add timestamp support, Kaidi is WIP.
+          GENERATE_AND_ADD_DATE_COLUMN(CiderDateType)
```

Comment:
refactor and unify this macro and call `GENERATE_AND_ADD_TIME_COLUMN(CiderDateType, DATE_DATA_GEN_STEP)`